src/include/migraphx/compile_options.hpp

```diff
@@ -11,6 +11,7 @@ struct compile_options
 {
     bool offload_copy = false;
     bool fast_math    = true;
+    bool enable_mlir  = false;
```

Comment:
I think for now we shouldn't need a flag. By default, migraphx doesn't use MLIR so this flag seems unnecessary. We can use the `MIGRAPHX_ENABLE_MLIR` build flag to decide if its enabled or not.

The problem with a flag like this is that none of the verification tests enable it. With a build time flag we can easily add a Jenkins stage with `-DMIGRAPHX_ENABLE_MLIR=On` and it will run all the verification tests with mlir enable. Once we get further along to use MLIR by default, then we can think about how we might want to enable or disable it.

Reply:
gone.